
geometry_msgs/Point fl
geometry_msgs/Point fr
geometry_msgs/Point rl
geometry_msgs/Point rr